canonical-web-and-design/ubuntu.com
# Update legal document ubuntu-advantage-service-terms

## Done
- Added point 4.6 to the ubuntu-advantage-service-terms document
- Fixed/improved the formatting on the document.

## QA
- Open the demo
- Visit /legal/ubuntu-advantage-service-terms
- Check the point 4.6 matches [the copy doc](https://docs.google.com/document/d/1-sraH8XHez4uDQy8FdK3bzTn_PIduzWJ-23lpnNrzMc/edit)
- Check the rendered page has no formatting issues

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/5763
